eda/src/autogluon/eda/analysis/__init__.py

```diff
@@ -1,3 +1,4 @@
 import phik
 from .base import Namespace
 from .dataset import Sampler
+from .shift import XShiftDetector
```

Comment:
nit: extra line at the end of file. Please run `flake8` in this package - it should highlight issues like this.